### PR TITLE
Percent stacked bar end axis no extra space

### PR DESCRIPTION
**Pull request by dippindots**
remove extra space for percent stacked bar chart
Fix # https://github.com/cBioPortal/cbioportal/issues/6020.

### CODE REVIEW

**Comment by kalletlak** [comment]
@dippindots you need to update screenshots

**Comment by dippindots** [comment]
@kalletlak Thanks! Will do!

File: src/shared/components/plots/MultipleCategoryBarPlot.tsx [hunk 1]
```diff
@@ -293,7 +293,7 @@ export default class MultipleCategoryBarPlot extends React.Component<IMultipleCa
 
     @computed get chartDomainPadding() {
         return {
-            [this.countAxis]:this.countAxisDomainPadding,
+            [this.countAxis]: this.props.percentage ? 0 : this.countAxisDomainPadding,
```
Comment (alisman) [inline src/shared/components/plots/MultipleCategoryBarPlot.tsx]: would it make sense to do this even if the axis is absolute (not percentage?)

Comment (dippindots) [reply src/shared/components/plots/MultipleCategoryBarPlot.tsx]: @alisman I found an example. Seems like we cannot do this to not percentage chart, right?
image

Comment (alisman) [reply src/shared/components/plots/MultipleCategoryBarPlot.tsx]: @schultzn @jjgao can you comment on the above?  With absolute values, should we always try to show a tick at the top?

Comment (jjgao) [reply src/shared/components/plots/MultipleCategoryBarPlot.tsx]: @alisman I think it's ok not to have a tick for absolute values.

**Pull request merged**